src/aequitas_cli/utils/configs_loader.py

```diff
@@ -5,22 +5,33 @@
 
 logging.getLogger(__name__)
 
+__author__ = "Rayid Ghani, Pedro Saleiro <saleiro@uchicago.edu>, Loren Hinkson"
+__copyright__ = "Copyright \xa9 2018. The University of Chicago. All Rights Reserved."
 
 class Configs(object):
+    original_fairness_measures = (
+        'Statistical Parity', 'Impact Parity', 'FDR Parity',
+        'FPR Parity', 'FNR Parity', 'FOR Parity', 'TPR Parity',
+        'Precision Parity')
+
     def __init__(self, ref_groups_method='majority', fairness_threshold=0.8,
-                 attr_cols=None, report=True, score_thresholds=None, project_description={'title': '', 'goal': ''},
-                 ref_groups=None, db=None, fairness_measures=['Statistical Parity', 'Impact Parity', 'FDR Parity',
-                                                              'FPR Parity', 'FNR Parity', 'FOR Parity', 'TPR Parity',
-                                                              'Precision Parity']):
+                 attr_cols=None, report=True, score_thresholds=None,
+                 project_description={'title': '', 'goal': ''},
```

Comment:
not part of your changes, but I see another mutable object which will be reused (and potentially modified) across instances of `Configs`. just FYI, this could instead be the immutable `tuple`:

    project_description=(('title', ''), ('goal', '')),

and then, regardless, the attribute `project_description` should be a newly-constructed `dict`:

    self.project_description = dict(project_description)

Reply:
@saleiro wondering if this particular variable might need to be updated after an instance is created (if a researcher forgets and then wants to name it, for example). Thoughts? 

And if so, @jesteria is there still a better/ best practice way to make sure the value can be updated if we try not to make mutable objects in signatures? I'm not as familiar with that scenario beyond setting up a getter/ setter.